### PR TITLE
Add shadow appearance

### PR DESCRIPTION
**Pull request by cozzin**
```swift
appearance.shadow = ToastShadow(color: UIColor.black.cgColor, opacity: 0.2, offset: CGSize(width: 0, height: 1), raidus: 10)
```

|  Without Shadow   |  With Shadow    |
|---    |---    |
|     |     |

### CODE REVIEW

File: Sources/ToastView.swift [hunk 1]
```diff
@@ -1,5 +1,31 @@
 import UIKit
 
+public class ToastShadow: NSObject {
+  public var color: CGColor?
+  public var opacity: Float?
+  public var offset: CGSize?
+  public var raidus: CGFloat?
+  public var path: CGPath?
+  
+  public init(color: CGColor? = nil, opacity: Float? = nil, offset: CGSize? = nil, raidus: CGFloat? = nil, path: CGPath? = nil) {
```
Comment (Coeur) [inline Sources/ToastView.swift]: Spelling: `raidus` -> `radius`

File: Demo/RootViewController.swift [hunk 2]
```diff
@@ -41,6 +41,7 @@ final class RootViewController: UIViewController {
     appearance.textInsets = UIEdgeInsets(top: 15, left: 20, bottom: 15, right: 20)
     appearance.bottomOffsetPortrait = 100
     appearance.cornerRadius = 20
+    appearance.shadow = ToastShadow(color: UIColor.black.cgColor, opacity: 0.2, offset: CGSize(width: 0, height: 1), raidus: 10)
```
Comment (Coeur) [inline Demo/RootViewController.swift]: Can we have some switch in the demo to activate/deactivate the shadow? I don't want to demo exclusively with a shadow, as it's not the default behavior.

File: Sources/ToastView.swift (hunk 1 above)
Comment (cozzin) [reply Sources/ToastView.swift]: Thanks, I changed at 8574794

File: Demo/RootViewController.swift (hunk 2 above)
Comment (cozzin) [reply Demo/RootViewController.swift]: Add 'toggle shadow' to example 1dd2298

**Comment by Coeur** [comment]
Thank you, but I'm still a bit hesitant to add a `class ToastShadow: NSObject`. I guess we're creating a `ToastShadow` because UIAppearance doesn't support customizing the `layer`, right? As it's only a container to store a few values, would it be doable as a struct or as a dictionary instead of a class? Are there any other useful values that could eventually justify to directly declare some larger `ToastLayerAppearance` instead of a limiting it to the shadow properties?

**Comment by cozzin** [comment]
@Coeur Actually, I wanted to make `ToastShadow` as `struct`, but I use `NSObject` for supporting objc.

**Comment by cozzin** [comment]
@Coeur I removed `ToastShadow` and add shadow properties to `ToastView`. Is is better? ea01579

**Comment by Coeur** [comment]
> Is is better?

Oh, yes, I like it better than your first draft.

**Comment by cozzin** [comment]
@Coeur Thank you for your approval 🙂 Is there anything else that needs to be changed?

**Comment by Coeur** [comment]
@devxoul could you also review the two simple pull requests #162 and #163 (this one) that I've approved? Having your thought on them could be nice too. And if approved, could we merge them and make a **2.3.0** release for Toaster? Thank you

**Comment by Coeur** [comment]
Well, no additional feedback, so I'll merge as is.

**Pull request merged**